### PR TITLE
Add a projected input option to point2dem

### PR DESCRIPTION
**Pull request by ScottMcMichael**
I am working with indoor point cloud scans and adding this feature allowed me to generate DEM files from them.  Not sure how broadly useful this is but I could add an example for it.

### CODE REVIEW

File: docs/tools/point2dem.rst [hunk 1]
```diff
@@ -397,6 +397,10 @@ Command-line options for point2dem
     files, if those files contain Easting and Northing fields. If
     not specified, ``--t_srs`` will be used.
 
+--input-is-projected
+    Treat the input coordinates as already in the projected coordinate
+    system, avoiding the need to convert to geodetic coordinates.
```
Comment (oleg-alexandrov) [inline docs/tools/point2dem.rst:402]: This should probably say: "avoiding the need to convert from ECEF to projected coordinates".

File: src/asp/Tools/point2dem.cc [hunk 2]
```diff
@@ -1296,23 +1298,29 @@ int main(int argc, char *argv[]) {
...
-        (asp::point_image_offset    // Add user coordinate offset
-         (asp::recenter_longitude   // XYZ to GDC, then normalize longitude
-          (cartesian_to_geodetic(point_image, output_georef), 
-           avg_lon),
-          Vector3(opt.lon_offset, opt.lat_offset, opt.height_offset)
-         ),
-         output_georef);
+
+    if (opt.input_is_projected) {
+      vw_out() << "\t--> Bypassing geodetic conversion!\n";
```
Comment (oleg-alexandrov) [inline src/asp/Tools/point2dem.cc:1303]: This can say: Assuming the input point cloud is already projected.

**Review by oleg-alexandrov — CHANGES_REQUESTED** [review]
I added two suggested changes. Note that "projected" is not the same as "geodetic". 

Also, it would help if this is in the NEWS.

File: docs/tools/point2dem.rst (hunk 1 above)
Comment (oleg-alexandrov) [reply docs/tools/point2dem.rst:402]: Or maybe it can be shortened to: "Avoiding the need to convert the points from ECEF."

Comment (ScottMcMichael) [reply docs/tools/point2dem.rst:402]: Sounds good.  I was referring to the point in the code where it gets converted to geodetic and back though I guess that is a hidden implementation detail.

**Pull request merged**